olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -76,6 +76,13 @@ file(GLOB GENERATED_HEADERS
 file(GLOB NETWORK_HEADERS
     "${CMAKE_CURRENT_SOURCE_DIR}/include/olp/core/network/*.h"
 )
+set(NETWORK2_HEADERS
+    "${CMAKE_CURRENT_SOURCE_DIR}/include/olp/core/network2/Network.h"
```

Comment:
replace network2 stub with final name?

Reply:
Discussion led to paths to be renamed in the last place after functionality is completed and done